README.md

```diff
@@ -2403,6 +2403,10 @@ in `conf/training/gpt3/5b.yaml`. The parameters can be modified to adjust the
 hyperparameters of the training runs. All supported model types and sizes can be found
 in `conf/training` folder.
 
+We support changeable global batch size during training. It can be set by changing `rampup_batch_size` parameter under the training config. Should be a list of 3 values: `[<start_batch_size>, <batch_size_increment>, <rampup_samples>]`<br>.
```

Comment:
```suggestion
We support  global batch size reampup during training. It can be set by changing `rampup_batch_size` parameter under the training config. Should be a list of 3 values: `[<start_batch_size>, <batch_size_increment>, <rampup_samples>]`<br>.
```